src/components/main.js

```diff
@@ -74,7 +74,11 @@ export default class Main extends Component {
 
   render() {
     var component = components[this.props.componentName || "Default"];
-
-    return h(component, this.state);
+    return (
+      <div>
```

Comment:
Instead of having to do the widgetContainer extension thing on every component, couldn't you add a class here and put it on this